### PR TITLE
Update dnn-model-services sample to use MPE

### PR DESCRIPTION
**Pull request by stellarspot**
- The DSL syntax is used to check and run shell commands
- The DNN sample is updated to use MPE
- The example-service sample is commented in feature file. It will be updated in the next pull request.

### CODE REVIEW

File: publish_dnn_model_services_test.go [hunk 1]
```diff
@@ -4,171 +4,96 @@ import (
...
 		"DAEMON_LISTENING_PORT": %s,
-		"ETHEREUM_JSON_RPC_ENDPOINT": "http://localhost:%s",
+		"DAEMON_END_POINT": "http://localhost:%s",
+		"ETHEREUM_JSON_RPC_ENDPOINT": "http://localhost:8545",
 		"PASSTHROUGH_ENABLED": true,
-		"PASSTHROUGH_ENDPOINT": "http://localhost:%s",
+		"PASSTHROUGH_ENDPOINT": "http://localhost:7003",
+		"IPFS_END_POINT": "http://localhost:5002",
+		"REGISTRY_ADDRESS_KEY": "%s",
+		"PRIVATE_KEY": "%s",
```
Comment (vsbogd) [inline publish_dnn_model_services_test.go]: Fake private key can be used here.

File: publish_dnn_model_services_test.go [hunk 2]
```diff
@@ -179,191 +104,36 @@ func dnnmodelServiceIsRunning() (err error) {
...
-
-	}
+	name := getTableValue(table, "name")
+	organization := getTableValue(table, "organization name")
+	daemonPort := getTableValue(table, "daemon port")
 
-	return
+	cmd := NewCommand().Dir(dnnModelServicesDir)
+	cmd.
+		Run("snet client balance --snt %s --multipartyescrow %s", singnetTokenAddress, multiPartyEscrow).
```
Comment (vsbogd) [inline publish_dnn_model_services_test.go]: I believe @astroseger mentioned that there is a way to not pass addresses of contracts manually each time.

File: publish_example_service_test.go [hunk 3]
```diff
@@ -45,7 +44,7 @@ func exampleserviceIsRunWithSnetdaemon(table *gherkin.DataTable) error {
     }`
```
Comment (vsbogd) [inline publish_example_service_test.go]: WIRE_ENCODING, SERVICE_TYPE, POLL_SLEEP, LOG_LEVEL are not required anymore

File: setup_environment_test.go [hunk 4]
```diff
@@ -329,6 +331,19 @@ func organizationIsAdded(table *gherkin.DataTable) (err error) {
 
 	organization := getTableValue(table, "organization")
 
+	return NewCommand().
+		Run("snet organization create %s --registry-at %s -y", organization, registryAddress).
+		Err()
+}
+
+func organizationIsAdded2(table *gherkin.DataTable) (err error) {
```
Comment (vsbogd) [inline setup_environment_test.go]: Are this function and organizationIsAdded used?

File: publish_dnn_model_services_test.go [hunk 5]
```diff
@@ -4,171 +4,96 @@ import (
...
-
-	return
+	daemonPort := getTableValue(table, "daemon port")
+	organization := getTableValue(table, "organization name")
+
+	metadata := dnnModelServicesDir + "/service_metadata.json"
+	cmd := NewCommand().Dir(dnnModelServicesDir)
+
+	cmd.
+		CheckFileContains(metadata, "display_name", displayName).
```
[the remark this answers is not in the document]
Comment (vsbogd) [reply publish_dnn_model_services_test.go]: Am I right that it is to set output file name before running command?
May be just use unique temporary file to collect command output?

File: publish_dnn_model_services_test.go [hunk 6]
```diff
@@ -4,171 +4,96 @@ import (
...
+		// TBD: convert organizationAddress to use right checksum
+		Run("snet service metadata_init service/service_spec \"%s\" %s --multipartyescrow %s",
+			displayName, "0x3b2b3C2e2E7C93db335E69D827F3CC4bC2A2A2cB", multiPartyEscrow).
```
[the remark this answers is not in the document]
Comment (AlexanderScherbatiy) [reply publish_dnn_model_services_test.go]: Yes. This is the address that is shown by Ganache in lower case and it should be converted to use right checksum. I am going to update it in the separated pull request.

File: publish_dnn_model_services_test.go (hunk 1 above)
Comment (AlexanderScherbatiy) [reply publish_dnn_model_services_test.go]: Ok. I will update this.

File: publish_dnn_model_services_test.go (hunk 2 above)
Comment (AlexanderScherbatiy) [reply publish_dnn_model_services_test.go]: It requires the following steps:
- migrate contracts
- get contract addresses and put the in networks folder in snet-cli
- rebuilding snet-cli
It seems that in this case running test and building process should be mixed.
I prefer to postpone this investigation to a separate pull request.

File: publish_example_service_test.go (hunk 3 above)
Comment (AlexanderScherbatiy) [reply publish_example_service_test.go]: I will remove the example-services from the test as obsolete.

File: publish_dnn_model_services_test.go (hunk 5 above)
[the remark this answers is not in the document]
Comment (AlexanderScherbatiy) [reply publish_dnn_model_services_test.go]: I will update this.

File: publish_dnn_model_services_test.go (hunk 1 above)
Comment (AlexanderScherbatiy) [reply publish_dnn_model_services_test.go]: I removed the example-service test.

File: publish_example_service_test.go (hunk 3 above)
Comment (AlexanderScherbatiy) [reply publish_example_service_test.go]: I removed the example-service test.

File: setup_environment_test.go (hunk 4 above)
Comment (AlexanderScherbatiy) [reply setup_environment_test.go]: I removed organizationIsAdded2 function.
organizationIsAdded functions is still used to add an organization.

File: publish_dnn_model_services_test.go (hunk 5 above)
[the remark this answers is not in the document]
Comment (AlexanderScherbatiy) [reply publish_dnn_model_services_test.go]: Updated.

**Pull request merged**